.gitattributes

```diff
@@ -1,4 +1,3 @@
 docs/* linguist-generated
 **/__snapshots__/** linguist-generated
 package-lock.json linguist-generated
-**/generated-snapshot.test.js
```

Comment:
I think you did the wrong thing - I think you want to remove the __snapshots__ line above instead!

Reply:
d'oh 🤦‍♂️